pkgs/development/interpreters/python/hooks/default.nix

```diff
@@ -109,6 +109,14 @@ in rec {
       };
     } ./setuptools-check-hook.sh) {};
 
+  venvShellHook = callPackage ({ }:
```

Comment:
`venv` has only been introduced in python 3.3, so maybe we should to something like `venvShellHook = disabledIf (! pythonAtLeast "3.3")` if we don't want to "automagically" support `virtualenv` for python 2.